wagtail/admin/views/generic/multiple_upload.py

```diff
@@ -154,6 +156,7 @@ def post(self, request):
             # Some other field of the form has failed validation, e.g. a required metadata field
             # on a custom image model. Store the object as an upload_model instance instead and
             # present the edit form so that it will become a proper object when successfully filled in
+            # self.object = self.save_object(form)
```

Comment:
looks like this added line is uninentional

Reply:
Yeah, already removed that.